src/js/dataobjects.js

```diff
@@ -21,6 +21,29 @@ export class Geography {
     }
 }
 
+export class IndicatorHelper {
+    static getMetadata(indicator) {
+        return indicator.metadata || {
+            source: "",
+            description: "",
```

Comment:
We should probably add ˋurlˋ here as well, as it is currently added

Reply:
I left it because it should be part of the source PR